src/cruise_control/scripts/cruise_control.py

```diff
@@ -57,9 +69,13 @@ def handle_speed_data(self, speed: Float64):
         rospy.loginfo("Handling the speed data")
         self.currentSpeed = speed.data
 
-    # Converts the PID controller value in the range of [0, 1) that the throttle and brake can have
+    # Converts the PID controller output value in the range of [0, 1) that the throttle and brake can have
     def pidToCarValues(self, inp):
-        return max(1 - math.exp(-2 * inp), 0.0)
+        # A lower value means a more gradual throttling and acceleration. However a higher input is needed to get a
+        # output value that is needed for a car to achieve its target speed than the PID controller will output. To deal
+        # with this, make the target speed higher than what is desired internally using self.speedAdjustmentFactor
+        exp_factor = -0.5
```

Comment:
exp_factor can have its value changed based off of what is observed to be better (have to change self.SpeedAdjustmentFactor as well in that case).

With factor = -0.5 I found that at higher speeds, say 7 m/s, there was less overshoot of the speed during the initial acceleration, but in the last turn our default test path it seemed to slow down rather heavily at times (to around 6.6 m/s). With factor = -2 the opposite was true. I'd say leave it for now and once we have more test paths maybe reconsider changing this value if we want.